cli/cli/main.go

```diff
@@ -49,25 +49,35 @@ func main() {
 		CallerPrettyfier:          nil,
 	})
 
-	if err := commands.RootCmd.Execute(); err != nil {
-		if !displayErrorMessageToCli(err) {
-			os.Exit(errorExitCode)
-		}
+	err := commands.RootCmd.Execute()
+	exitCode := executeCommand(err)
+	os.Exit(exitCode)
+}
 
-		maybeCleanedError := out.GetErrorMessageToBeDisplayedOnCli(err)
-		errorMessageFromCli := maybeCleanedError.Error()
+func executeCommand(err error) int {
```

Comment:
I think that this method should be renamed, right? because it's not executing commands, is more like `handling the execution error`